deepchecks/nlp/utils/text_properties.py

```diff
@@ -260,14 +297,24 @@ def language(
     return language_codes
 
 
-def sentiment(raw_text: Sequence[str]) -> List[str]:
+def sentiment(raw_text: Sequence[str]) -> List[float]:
     """Return list of floats of sentiment."""
-    return [textblob.TextBlob(text).sentiment.polarity for text in raw_text]
+    if text_blob_cache.get('textblob') is None:
+        # TextBlob uses only the words and not the relations between them, so we can sample the text
```

Comment:
So sampling is a good idea, but should really replace textblob...

Reply:
Why? And do you have an alternative?